packages/ember/addon-test-support/index.js

```diff
@@ -0,0 +1,7 @@
+import { getContext } from '@ember/test-helpers';
+
+export function getWorker(name) {
```

Comment:
maybe this'd be TS' `getWorker<K extends keyof WorkerRegistry>(name: K): WorkerRegistry[K]`?

Reply:
I haven't set up ts support for the ember portions of this project just yet, but yeah that would be something nice :)